README.md

```diff
@@ -175,3 +179,47 @@ Finally, write a new changelog file in `api/db/changelog/` and include it in
 `db.changelog-master.xml`.
 
 `liquibase` does not roll back partially failed changes.
+
+## Docker Troubleshooting
+
+### Permissions errors
+
+Always `export UID` (or place that in your `.bashrc`) before running
+`docker-compose` commands outside of `project.rb` (which takes care of setting
+`UID`).
+
+If you get permissions errors like
+`Exception in thread "main" java.io.FileNotFoundException: /.gradle/wrapper/dists/gradle-4.0-all/ac27o8rbd0ic8ih41or9l32mv/gradle-4.0-all.zip.lck (Permission denied)` or from MySQL like
+`ERROR 1130 (HY000): Host '172.18.0.3' is not allowed to connect to this MySQL server`
+it may be because you ran a `docker` command as root (the default) whereas
+our `project.rb` scripts run as you. To clear permissions in Docker volumes,
+remove the volumes:
+
+```
+docker volume rm $(docker volume ls -q)
```

Comment:
Maybe change this to only `api_db` and/or workbench volumes. `docker volume rm $(docker volume ls -q | grep api_)` ?

Reply:
Hey Mark, If i remember correctly this was the error i was getting: Exception in thread "main" java.io.FileNotFoundException: /.gradle/wrapper/dists/gradle-4.0-all/ac27o8rbd0ic8ih41or9l32mv/gradle-4.0-all.zip.lck (Permission denied). Seems that ./project docker-clean fixed my problems. I don't have very much experience with docker, so i'm not exactly sure why i was getting this error.